packages/editor-ui/src/views/SettingsUsersView.vue

```diff
@@ -203,7 +203,10 @@ export default defineComponent({
 		},
 		async onDisallowSSOManualLogin(userId: string) {
 			const user = this.usersStore.getUserById(userId) as IUser | null;
-			if (user?.settings) {
+			if (user) {
```

Comment:
I believe this change has been merged already in another PR

Reply:
yes, and it was already in master. weird that it did not merge in correctly. will do it manually.